index.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./plugin");
```

Comment:
This is redundant with our `main` in our package.json. Either update the package.json if this is needed or remove this.

Reply:
Is it? I never understand the `main` in `package.json`